backend/clickhouse/log_row.go

```diff
@@ -14,6 +15,8 @@ import (
 	log "github.com/sirupsen/logrus"
 )
 
+const LogAttributeValueLengthLimit = 2 << 15
```

Comment:
Can we set to a lower limit now? I pulled a random stacktrace and it was 3455 characters ([source](https://go.dev/play/p/4FNtQ-cd0Sh)) so 64K characters is far too lenient.

I don't see a reason why anyone needs more than 500 characters. And we can choose to simply trim (instead of drop) those attributes if someone complains. 

Particularly, I think we should put a more aggressive guard in now given that our pricing model is based on number of rows and not by storage.

Reply:
added a limit on attribute values and the body as per https://highlightcorp.slack.com/archives/C04NUD29BT3/p1679944459103359